src/service/platform-impl.js

```diff
@@ -98,6 +98,15 @@ export class Platform {
   }
 
   /**
+   * Whether the current browser is isStandAlone.
+   * @return {boolean}
+   */
+  isStandAlone() {
+    return /iPhone|iPad|iPod/i.test(this.navigator_.userAgent) &&
```

Comment:
`navigator.standalone` is only defined in iOS, so we can simplify this to just return the value of the property without checking if it exists, since `undefined` is falsy.